stream/src/placeholder/wire.proto

```diff
@@ -14,7 +14,7 @@ package wire;
 message Handshake {
     bytes recipient_public_key = 1;
     bytes ephemeral_public_key = 2;
-    uint64 timestamp = 3;
+    uint64 timestamp_ms = 3;
```

Comment:
nit: IMO we can just leave this as `timestamp`

Reply:
I kind of like having the units be included since the `Duration` in rust includes nanoseconds as the smallest value, and previously the timestamp was in seconds